### ISSUE TITLE
[Feature Request] value as a language studies aid

### ISSUE THREAD

**Issue by Pantyhose-X** [nvaccess/nvda#12545]
### Is your feature request related to a problem? Please describe.
- NVDA to talk not to read the screen

Applied to language learner. Language worker's
Applied to language learning. Pronunciation correction

### Describe the solution you'd like
`We only need to read the keyboard out loud and not anything else aloud.`  [NVDA](https://github.com/nvaccess/nvda) cannot do that.
l only want NVDA to talk when you press keys on the
keyboard, not to read the screen
![C](https://user-images.githubusercontent.com/42504385/122143713-1957a780-cea6-11eb-9415-ba6fe179d345.JPG)

### Additional context
Use alone keyboard Reader

https://user-images.githubusercontent.com/42504385/132130706-a1a8ec10-416a-4711-b5d3-c91086e9710f.mp4

https://user-images.githubusercontent.com/42504385/127757873-c3c076d3-ce73-40b1-8f57-423459fc0c2f.MP4

**Comment by XLTechie** [comment]
@Pantyhose-X Please try describing the behavior you want and expect. Your 
video does not clarify the issue, and your little one line description 
doesn't make sense, at least to me.

Try explaining what you do, what happens now, and what you want to have 
happen.

**Comment by Pantyhose-X** [comment]
> @Pantyhose-X Please try describing the behavior you want and expect. Your video does not clarify the issue, and your little one line description doesn't make sense, at least to me. Try explaining what you do, what happens now, and what you want to have happen.

Turn off mouse tracking read aloud

**Comment by XLTechie** [comment]
Try pressing NVDA+M.

**Comment by Pantyhose-X** [comment]
> Try pressing NVDA+M.

NVDA is that button ?

**Comment by XLTechie** [comment]
When someone tells you to press NVDA and another button, it is whatever 
key you have setup to be the one that NVDA responds to.

This is usually insert, numpad insert, capslock, or maybe any one of them.

You might want to read the keyboard quick reference (In the NVDA help 
menu).

Another way to turn off mouse tracking is to go into the NVDA menu, 
preferences, settings, mouse, and turn off mouse tracking there (second 
checkbox).

You should subscribe to one of the NVDA user mailing lists. There is 
probably one for your language available, or you can subscribe to the 
general English one at https://groups.io/g/nvda

Lists are a better place for usage questions. These issues are for 
reporting bugs, and requesting features.

**Comment by Pantyhose-X** [comment]
> When someone tells you to press NVDA and another button, it is whatever key you have setup to be the one that NVDA responds to. This is usually insert, numpad insert, capslock, or maybe any one of them. You might want to read the keyboard quick reference (In the NVDA help menu). Another way to turn off mouse tracking is to go into the NVDA menu, preferences, settings, mouse, and turn off mouse tracking there (second checkbox). You should subscribe to one of the NVDA user mailing lists. There is probably one for your language available, or you can subscribe to the general English one at https://groups.io/g/nvda Lists are a better place for usage questions. These issues are for reporting bugs, and requesting features.

* I just need to use ‘keyboard’
* How do I turn off other read aloud?
In the NVDA settings I do not use the functions, I have turned them off, but still read aloud, content other than the keyboard

**Comment by XLTechie** [comment]
Pantyhose-X wrote:

>  *  I just need to use ‘keyboard’
>  *  How do I turn off other read aloud?
>     In the NVDA settings I do not use the functions, I have turned them off, but still read aloud, content other than the keyboard

Are you saying that you only want NVDA to talk when you press keys on the 
keyboard, not to read the screen?

**Comment by Pantyhose-X** [comment]
> Pantyhose-X wrote:
> * I just need to use ‘keyboard’ * How do I turn off other read aloud? In the NVDA settings I do not use the functions, I have turned them off, but still read aloud, content other than the keyboard
> Are you saying that you only want NVDA to talk when you press keys on the keyboard, not to read the screen?

Yes not to read the screen

**Comment by alexarnaud** [comment]
@Pantyhose-X I also have some trouble to understand what is the problem you're facing. Don't worry, it's not ready to report issues when you're not familiar with it.

To help us help you, I suggest you to tell us a steps by steps process :
1) Do X
2) Do Y
3) Do Z

Result:
I have this result.

Expected result:
I would like this result.

The reason why we're requesting you a very precise steps to follow is because we don't have this problem ourselves and honestly your video doesn't most of the people here which are blind.

If providing steps to reproduce is not possible, could you give us an detailed exemple of something you don't want to be read by NVDA.

**Comment by XLTechie** [comment]
This can not be done. Someone would need to write an add-on for you.

I think you may want something other than a screen reader.

**Comment by Pantyhose-X** [comment]
> This can not be done. Someone would need to write an add-on for you. I think you may want something other than a screen reader.


# Use this feature alone[![Watch the video](https://user-images.githubusercontent.com/42504385/122677974-e2341e00-d238-11eb-9a10-f7fd00ffb7d9.JPG)](https://www.youtube.com/watch?v=YPBn6D6pf78&t=213s)

**Comment by ehollig** [comment]
It sounds as if the author is asking for something specific that can be performed by another program than a screen reader. Closing this issue
